OxfordBodleian/Juel-Jensen/BDLaethg36.xml

```diff
@@ -102,90 +95,304 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                  A yellow thread emerges from the tail edge of the bookblock.
+                                    A piece of folded parchment protects the spine and is linked to the bookblock by stitching it to the holes for board attachment and the chains that the sewing forms on the spine.
+                                    A yellow thread emerges from the tail edge of the bookblock.
                                 </decoNote>
                             </binding>
                         </bindingDesc>
-
+                        
                     </physDesc>
 
-                <!--    <history>
-                        <origin>
-                            <origDate when="" evidence=""/>
-                        </origin>
-                        <provenance>  </provenance>
-                    </history>-->
+                    <history>
+                        <provenance>Unknown provenance. According to <ref target="#e1">Additional note 1</ref>, the manuscript was
+                            bought by Mesraq in <placeName ref="#LOC1216AddisA"/> and given to <persName ref="PRS5782JuelJen"/> before
+                            1950. <ref target="#p2">Unit 2</ref> is a single leaf sown onto a leaf in <ref target="#p1">Unit 1</ref> at an
+                            unknown point in time.
+                        </provenance>
+                    </history>
+                    
+                    
+                    <msPart xml:id="p1">
+                        <msIdentifier>
+                            <idno/>
+                        </msIdentifier>
+
+                    <msContents>
+                        <summary/>
+
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="40r"/>
+                            <title type="complete" ref="LIT2246salotz"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez" type="inscriptio">
+                                <locus target="#1r"/>
+                                <hi rend="rubric">በስመ፡ <sic>አብብ፡</sic> ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡</hi> <del rend="erasure"/> 
+                                ጸሎ<add place="above">ት</add>፡ ዘእግዝእትነ፡ <hi rend="rubric">ማርያም<supplied reason="omitted">፡</supplied></hi> 
+                                ወላዲተ፡ አምላክ፡ ቅድስት፡ ድንግል<supplied reason="omitted">፡</supplied> እሙ፡ ለብርሃን፡ እንተ፡ ጸሐፋ፡ 
+                                አብሮክሮስ<supplied reason="omitted">፡</supplied> ረድኣ፡ ዮሐንስ፡
+                            </incipit>
+                            <incipit xml:lang="gez" type="supplication">
+                                <locus target="#1r"/>
+                                ጸሎት፡ ወበረከታ፡ ወኃይለ፡ ረድኤታ፡ የሀሉ፡ ምስለ፡ ነፍሰ፡ አቡነ፡ ገብረ፡ <del rend="erasure"/>
```

Comment:
I think this person should be marked up

Reply:
Could you give me a hint on how you normally encode names that are only fragmentarily attested? Like of you only have "Gabra" and the rest is erased. @thea-m @MassimoVilla

Reply:
Thank you!